Jellyfin.Networking/Manager/NetworkManager.cs

```diff
@@ -691,11 +691,11 @@ private void ConfigurationUpdated(object? sender, ConfigurationUpdateEventArgs e
         /// Checks the string to see if it matches any interface names.
         /// </summary>
         /// <param name="token">String to check.</param>
-        /// <param name="index">Interface index number.</param>
+        /// <param name="index">Interface index numbers that match.</param>
         /// <returns><c>true</c> if an interface name matches the token, <c>False</c> otherwise.</returns>
-        private bool IsInterface(string token, out int index)
+        private bool IsInterface(string token, out List<int>? index)
```

Comment:
Why a nullable list? Shouldn't it be an `IReadOnlyList<int>`? In the case of no results it should be empty. Null feels bad.

Reply:
Renamed the function to TryIsInterface and added the [NotNullWhen(true)]  attribute on the method.

Used null as its more optimised - no empty list needs to be created. Likewise with IReadOnlyList, as it's a private method and only used here)

Reply:
Enumerable. Empty is static. There's no allocation.

Reply:
Looked at the code returning an Enumerable.Empty would add a level of complexity.

The current setup mimics the TryXXXXX design whereby if the method returns _true_ then _index_ has a value, if _false_ then _index_ is _null_.

Reply:
The code would be a little simpler with Enumerable.Empty actually, but I get your point about mimicking other Try-function behaviours.

Could you find a better name though? `TryIsInterface` doesn't really say anything meaningful about the intent.

Reply:
Renamed to TryGetInterfaces